### PR TITLE
#48 Landing Page

### PR DESCRIPTION
**Pull request by brittanygay**
connect #48 

- Stubbed out the Landing page 

**Note**:
The typeface isn't working locally for some reason but it should be working on staging. Once I can view on staging I may make a new PR with some additional typeface tweaks

**Note to @Mcscott142**:
We still need some copy to add, we could add this later but I would rather get it in as soon as possible so we don't forget.

**List of other needs**:
- [ ] Glamourshot of product (if this is not do-able since we dont even have the product built out yet, we can remove it)
- [ ] logos (if we need to add any)
- [ ] Two images for the highlights section (I can work from what we already have for priority)
- [ ] The rest of the copy

screencapture-localhost-4000-2018-08-03-12_18_22
screencapture-localhost-4000-2018-08-03-12_18_36

### CODE REVIEW

File: assets/css/components/_Footer.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  padding: $global-padding;
+  padding-bottom: 0;
+  @include breakpoint(medium) {
+    @include flex(center, row, center);
+  }
+  &-logo,
+  img {
+    display: inline-block;
+    margin-left: $global-padding-small;
+    max-width: 10rem !important;
```
Comment (lintomat) [inline assets/css/components/_Footer.scss]: !important should not be used

File: assets/css/components/_Footer.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.FooterWrapper {
+  background: $black-40;
+  &.img {
+    background-image: url('/images/footer-bkgd.jpg');
```
Comment (lintomat) [inline assets/css/components/_Footer.scss]: Properties should be ordered background-blend-mode, background-color, background-image, background-repeat, background-size, clear

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/84).

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/84).

File: lib/incentivize_web/templates/page/index.html.eex [hunk 3]
```diff
@@ -1 +1,115 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="Section">
+      <div class="rev-Row">
+        <div class="rev-Col rev-Col--medium8 rev-Col--smallCentered Text-center">
+          <h2>About Incentivize</h2>
+          <p>The software industry is lacking an <strong>incentive platform</strong> for building developer communities around projects outside of bounty offerings, employers paying developers hourly/salaries, or maintainers receiving donations. Contributors to open and closed source projects should be incentivized for the value they bring to a project beyond just their time.</p>
```
Comment (Mcscott142) [inline lib/incentivize_web/templates/page/index.html.eex]: <p>The software industry is lacking an <strong>incentive platform</strong> for building communities around projects outside of bounty offerings, employers paying hourly/salaries, or maintainers receiving donations. Contributors to open and closed source projects should be incentivized for the value they bring to a project beyond just their time.</p>

Comment (Mcscott142) [reply lib/incentivize_web/templates/page/index.html.eex]: Looks good I just removed the word developer.

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/84).

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/84).

**Comment by brittanygay** [comment]
@Mcscott142 I revised a little of the copy, and updated the last section with some new copy, and I also hid the logos section for now. 

@blazebarsamian I put in some typography tweaks just now. Here are screenshots with the updated screen

Ready for yall to review once more! Sorry about that!

screencapture-localhost-4000-2018-08-03-13_16_40
screencapture-localhost-4000-2018-08-03-13_20_07

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/84).

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/84).

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/84).

**Comment by Mcscott142** [comment]
Looks good. I like it 👍

**Pull request merged**